.npmrc

```diff
@@ -0,0 +1 @@
+shamefully-hoist=true
```

Comment:
what is the reason of having this?

Reply:
This helps me to reduce changes to the current structure of the repository, which I try to follow as much as possible.

`shamefully-hoist` creates a symlink imprint of each dependency in the root `node_modules/*`. It helps any package access real dependency (in `node_modules/.pnpm/*`) without declaring it in local `package.json`. So, it generally allows dependency hoist read, which is used in the repo a lot.

Without it, I would need to make extra changes:
- Make sure each package specifies all dependencies it uses.
- Conver `__test__` folder to a package.
- Remove TS config option, which resolves default pnpm dependencies location.

```
    "paths": {
      "*": [
        "node_modules/*"
      ]
    },
```

Reply:
So do you want to keep `shamefully-hoist=true` or perform these extra changes?
For me both options are fine.